src/index.js

```diff
@@ -41,6 +46,11 @@ program
     /^(class|pure-class|functional)$/i,
     config.type
   ).option(
+    '-s, --styling <stylingSolution>',
+    'Which styling solution to generate for (default: "none")',
```

Comment:
Copy nit: Can we make this `Styling tool to use for the component (default: "none")`

Reply:
Sure! You know English better than me for SURE :)